src/compiler/utilities.ts

```diff
@@ -1669,6 +1669,11 @@ namespace ts {
         return false;
     }
 
+    export function isFunctionDeclarationIdentifierName(node: Identifier): boolean {

```

Comment:
it doesn't seem to be use outside " src/services/services.ts", it should be move there since other similar helper functions are there as well